js/stores/appStore.js

```diff
@@ -503,7 +503,7 @@ const handleAppAction = (action) => {
           isDefaultBrowser = app.isDefaultProtocolClient('', desktopName)
         } else {
           for (const p of defaultProtocols) {
-            app.setAsDefaultProtocolClient(p)
+            app.setAsDefaultProtocolClient(p, ['--'])
```

Comment:
doesn't the 2nd argument (`path`) need to be set? https://electronjs.org/docs/api/app#appsetasdefaultprotocolclientprotocol-path-args-macos-windows

Reply:
it uses default value `Defaults to process.execPath`

Reply:
we should add `['--']` just for Windows just like `protocolHandlerMigration` did